### DISCUSSION TITLE
How to disable `volar` activation on `markdown` files?

### DISCUSSION THREAD

**Discussion by sabertaz** [vuejs/language-tools#1430] [Q&A] [answered]
I don't use `vitepress`, just prefer original `GFM` format.
Suggestions provided by `volar` disturb my `markdown` writing.

Currently, I disable whole extension when editing `markdown` files,
then enable extension again when editing `vue` files.

Is there any better solution?

**Comment by johnsoncodehk** [comment, marked as answer]
Fixed in https://github.com/johnsoncodehk/volar/commit/c29bdb2b374faaf5d70e1d2a3d8faf9b7d96846c, and in next version you can config control it by `include`, `exclude` in tsconfig.
